docs/dapp/sapphire/gsn.md

```diff
@@ -0,0 +1,78 @@
+---
+description: CLI instructions for deploying GSN on Sapphire Testnet
+---
+# Gas Station Network
+
+[Gas Station Network](https://docs.opengsn.org) is very useful for improving user experiences of many Dapps by making their users send transactions without having native tokens for gas. We make GSN functionalities available on Sapphire Paratime.
```

Comment:
There is a standard "vocabulary" for terms such as dApp and ParaTime so they are consistent across the docs: https://github.com/oasisprotocol/docs/blob/main/README.md#vocabulary

```suggestion
[Gas Station Network](https://docs.opengsn.org) is very useful for improving user experiences of many dApps by making their users send transactions without having native tokens for gas. We make GSN functionalities available on Sapphire ParaTime.
```